KeePass warns HIBPOfflineCheck.dll is incompatible with the current KeePass version

KeePass version: 2.41
HIBPOfflineCheck version: 1.5.0.0

I'm able dismiss the warning three times and am then able to open KeePass and use the HIBP plugin as expected.

Here is the full message:
Keepass 

The following plugin is incompatible with the current Keepass 
version: 
C:\Users\*myprofile*\AppData\Local\KeePass\HIPBOfflineCheck.dll 

Have a look at the plugin's website for an appropriate version. 

Value cannot be null. 
Parameter name: prov 
Keepass 
 at KeePass.UI.ColumnProviderPool.Remove(ColumnProvider prov) 
at HIBPOfflineCheck.HIBPOfflineCheckExt.Terminate() 
at HIBPOfflineCheck.HIBPOfflineCheckExtCheckExt.Initialize(IPIuginHost host) 
at KeePass.Plugins.PluginManager.LoadPlugins(List'1 IFiIes, String strTypeName, String strDisplayFilePath, Boolean bSkipCacheFiIes) Boolean Remove(KeePass.UI.ColumnProvider)

### For documentation

The compiled plugins can be found here:

```
~/.local/share/KeePass/PluginCache/
```

### Debugging

Thanks for the link it says:

```
PLGX debugging. When the command line option --debug is passed and a PLGX plugin fails to compile, the output of all tried compilers is saved to a temporary file.
```
* I removed the compiled dll 
* I removed package `mono-devel` and all its dependencies
* I added `--debug` 

A log file was created. It says:

```
error CS0006: Metadata file `System.DirectoryServices.dll' could not be found
```

--- a/readme.md
+++ b/readme.md
@@ -32,6 +32,8 @@ The plugin adds a new column to KeePass. When double-clicking the column for a s
 
 If you are using KeePass on Windows, this plugin requires **Microsoft .NET Framework 4.5** to run.
 
+When running KeePass under Mono (on Ubuntu/Debian), if the plugin does not compile or load, verify that you have installed the `mono-mcs` package. This package contains the compilers needed for KeePass to compile the plugins in the PLGX format.
+
 - Download the latest version of the [password list](https://haveibeenpwned.com/Passwords) file from the _haveibeenpwned.com_ website. The required file is the one with format **SHA-1 (ordered by hash)**. Use the _Pwned Passwords downloader_ (as of May 2022, this is the best way to get the most up-to-date passwords as suggested by the _Have I Been Pwned_ project's author). Alternatively, use the torrent method.
 
     __It's important that you get the SHA-1 (ordered by hash) version of the file, the plugin uses it for fast searching__.

